backend/picture_muvie/utils/naver_search.py

```diff
@@ -32,20 +32,20 @@ def __call__(self, *args):
         return self.value[2](*args)
 
 
-def get_lyrics(title: str = "", artist: str = "") -> tuple[str]:
+def get_lyrics(title: str = "") -> tuple[str]:
```

Comment:
여기 밑 부분도 전부 다 query로~
```suggestion
def get_lyrics(query: str = "") -> tuple[str]:
```